checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallInferenceLogic.java

```diff
@@ -52,95 +94,621 @@ public class MustCallInferenceLogic {
    * @param cfg the ControlFlowGraph
```

Comment:
I think this is the CFG for a single method.  Please document that (or document what it is, if I'm wrong).

Reply:
I updated the comment.